vpc-sample/Dockerfile

```
@@ -0,0 +1,14 @@
+# Use the official Python image.
+# https://hub.docker.com/_/python
+FROM python:3.8-slim
+
+# Copy local code to the container image.
+ENV APP_HOME /app
+WORKDIR $APP_HOME
+COPY . .
```

Prefer directory target to avoid confusing docker errors if a developer modifies the Dockerfile for multiple sources.

```suggestion
COPY . ./
```